pkgs/development/libraries/qt-5/5.15/qtbase.patch.d/0014-qtbase-gcc11-fix.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/concurrent/qtconcurrentthreadengine.h b/src/concurrent/qtconcurrentthreadengine.h
+index cbd8ad04..4cd5b85 100644
+--- a/src/concurrent/qtconcurrentthreadengine.h
++++ b/src/concurrent/qtconcurrentthreadengine.h
+@@ -256,8 +256,8 @@
+ class ThreadEngineStarter<void> : public ThreadEngineStarterBase<void>
+ {
+ public:
+-    ThreadEngineStarter<void>(ThreadEngine<void> *_threadEngine)
+-    :ThreadEngineStarterBase<void>(_threadEngine) {}
++    ThreadEngineStarter(ThreadEngine<void> *_threadEngine)
++        : ThreadEngineStarterBase<void>(_threadEngine) {}
```

Comment:
Can we fetchpatch https://codereview.qt-project.org/gitweb?p=qt/qtbase.git;a=patch;h=049e14870c13235cd066758f29c42dc96c1ccdf8 instead?

Reply:
I originally wanted to do that as well, but wasn't sure how to fetch the diff without going through their download form. Thanks for the link! I've commited your suggestion :+1: